README.md

```diff
@@ -1,25 +1,11 @@
 # level-community
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
+> Gathers common information related to projects in the `Level` community
 
-**Gathers common information related to projects in the `Level` community**
+[![level badge][level-badge]](https://github.com/level/awesome)
 
 When creating a new project. Either link to this repository or include it as a dependency in `package.json`.
 
-### License &amp; Copyright
-
-Copyright (c) 2012-2015 `Level` contributors.
-
-`Level` projects are licensed under the MIT license. All rights not explicitly granted in the MIT license are reserved. See the included [`LICENSE.md`](https://github.com/level/community/blob/master/LICENSE.md) file for more details.
-
-### Contributing
-
-`Level` projects are **OPEN Open Source Projects**. This means that:
-
-> Individuals making significant and valuable contributions are given commit-access to a project to contribute as they see fit. A project is more like an open wiki than a standard guarded open source project.
-
-See the [`CONTRIBUTING.md`](https://github.com/level/community/blob/master/CONTRIBUTING.md) file for more details.
-
```

Comment:
I moved these sections to the bottom of `### Contributors` since that information should be at the top.